web/thesauruses/_meta/io.json

```diff
@@ -0,0 +1,243 @@
+            "code": ""
+        },
+        "write_line_with_new_line": {
+            "name": "Write a line to the console with a return charactor",
+            "code": ""
+        },
+        "read_line": {
+            "name": "Read a string of user input from the console",
+            "code": ""
+        },
+        "read_char": {
+            "name": "Read a single char of user input from the console",
+            "code": ""
+        },
+        "clear_console_cutput": {
+            "name": "Clear the console output",
+            "code": ""
+        },
+        "change_console_color": {
+            "name": "Change the color of the console",
+            "code": ""
+        },
+        "list_directory": {
+            "name": "List the items in the current directory",
+            "code": ""
+        },
+        "read_directory_info": {
+            "name": "Get infomation about the current directory (number of items, size, etc)",
+            "code": ""
+        },
+        "create_directory": {
+            "name": "Create a directory",
+            "code": ""
+        },
+        "delete_directory": {
+            "name": "Delete a directory",
+            "code": ""
+        },
+        "rename_directory": {
+            "name": "Rename a directory",
+            "code": ""
+        },
+        "move_directory": {
+            "name": "Move a directory to another locaiton",
+            "code": ""
+        },
+        "update_directory_permissions": {
+            "name": "Update the permissions a user, group, or application has to a directory",
+            "code": ""
+        },
+        "read_line_of_file": {
+            "name": "Read a single line from a file",
+            "code": ""
+        },
+        "read_lines_of_file": {
+            "name": "Read all the lines in a file",
+            "code": ""
+        },
+        "read_file_stream": {
+            "name": "Read the file into a stream",
+            "code": ""
+        },
+        "create_file": {
+            "name": "Create a file",
+            "code": ""
+        },
+        "delete_file": {
+            "name": "Delete a file",
+            "code": ""
+        },
+        "update_file": {
+            "name": "Update a file",
+            "code": ""
+        },
+        "move_file": {
+            "name": "Move a file to a new location",
+            "code": ""
+        },
+        "send_a_get_request": {
+            "name": "Send a HTTP GET request",
+            "code": ""
+        },
+        "send_a_post_request": {
+            "name": "Send a HTTP POST request",
+            "code": ""
+        },
+        "add_headers": {
+            "name": "Add headers to a HTTP request",
+            "code": ""
+        },
+        "add_query_params": {
+            "name": "Add query parameters to a HTTP request",
+            "code": ""
+        },
+        "inbuilt_url_class": {
+            "name": "The inbuilt URL class",
```

Comment:
What do you mean by this? Do you have an example in some language to what you're referring to?

Reply:
In C# under system there is a Uri class, in Java it is URI under java.net, Typescript and I think now JS have a URI class implementation as well. All used as a standard way to handle URL/Is in a standard way and to validate them against respective standards.

Reply:
Ah, ok, this makes sense now. Thanks!